MagicalCryptoWallet.Tests/NodeBuilding/CoreNode.cs

```diff
@@ -144,6 +146,7 @@ public Node CreateNodeClient(NodeConnectionParameters parameters)
 			{
 				{"regtest", "1"},
 				{"rest", "1"},
+				{"listenonion", "0"},
```

Comment:
@lontivero Why is it needed?

Reply:
The log file was full of Tor related lines that made it a bit annoying to read. That's what I switch it off.